targets/UnrealMarketplacePlugin/source/PlayFab/Source/PlayFabCpp/Public/PlayFab.h.ejs

```diff
@@ -20,6 +21,17 @@ namespace PlayFab
             Instance->ClearFlags(RF_Standalone);
         });
     }
+#else
+    template<typename T>
+    TSharedPtr<T> MakeSharedUObject()
+    {    
+        return TSharedPtr<T>(NewObject<T>((UObject*)GetTransientPackage(), NAME_None, RF_StrongRefOnFrame),
```

Comment:
It seems like the RF_Standalone vs RF_StrongRefOnFrame is the only line-difference between the WITH_EDITOR vs not.
I would prefer to only #If/#else the one line that's different (for now).